controls/scheduler/customdateformats.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Custom Date Formats
+page_title: .NET MAUI Scheduler Documentation - Custom Date Formats
+description: Learn more about the custom date formats feature in the Telerik UI for .NET MAUI Scheduler control.
+position: 11
+slug: scheduler-custom-date-formats
+---
+
+# Custom Date Formats
+
+The Telerik UI for .NET MAUI Scheduler control provides built-in date formatting support. Each Scheduler view can be easily customized to display dates and times on the time ruler and in the headers in custom formats.
+
+You can use the following date format properties for the different views:
+
+### MultiDay View
```

Comment:
According to the Views documentation, we decided to go with `Multiday` as the spelling.

```suggestion
### Multiday View
```